src/serac/numerics/functional/detail/hexahedron_H1.inl

```diff
@@ -96,8 +96,8 @@ struct finite_element<Geometry::Hexahedron, H1<p, c> > {
   template <bool apply_weights, int q>
   static constexpr auto calculate_B()
   {
-    constexpr auto                  points1D  = GaussLegendreNodes<q>();
-    [[maybe_unused]] constexpr auto weights1D = GaussLegendreWeights<q>();
+    constexpr auto                  points1D  = GaussLegendreNodes<q, mfem::Geometry::SEGMENT>();

```

Comment:
Is it possible to rename `q` to `q1D`? As you can tell by my comments, this has bitten me in the past.

Reply:
Q1D makes sense for tensor product elements, but there is no underlying 1D quadrature rule for simplices

Reply:
oops, I think I misunderstood the comment: maybe you meant for just the tensor product elements?

Reply:
yes - I believe my thought was this is only applicable to the tensor product elements. Really, anything to help clarify what `q` means in it's local context is appreciated.